packages/thirdweb/src/react/web/ui/ConnectWallet/screens/ViewNFTs.tsx

```diff
@@ -145,42 +80,30 @@ export function ViewNFTsContent(props: {
           gap: "12px",
         }}
       >
-        {results.map((result, index) => {
-          if (result.error) {
-            console.error(result.error);
-            return null;
-          }
-          return result.isLoading || !result.data ? (
-            <Skeleton
-              key={`${nftList[index]?.chain?.id}:${nftList[index]?.address}`}
-              height="150px"
-              width="150px"
-            />
-          ) : (
-            result.data.map((nft) => (
-              <NftCard
-                key={`${nft.chain.id}:${nft.address}:${nft.id}`}
-                {...nft}
-                client={props.client}
-                chain={nft.chain}
-                theme={props.theme}
-              />
-            ))
-          );
-        })}
+        {isLoading && <Skeleton height="150px" width="150px" />}
```

Comment:
We should show more than one skeleton while loading

Reply:
But we don't know how many NFTs the user has before we load them?